website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -305,7 +304,6 @@ of several sub-resources - these resources are laid out below.
     whether the object has been updated. Defaults to 0 seconds.
 
 * `origin_request_policy_id` (Optional) - The unique identifier of the origin request policy
-    that is attached to the behavior.
```

Comment:
Was this intentionally removed? Due to being unfamiliar, I can't quite tell if this bit of the sentence is important, or was removed because it's not.